src/reanimated2/component/FlatList.tsx

```diff
@@ -91,7 +92,7 @@ const FlatListForwardRefRender = function <Item = any>(
     // @ts-expect-error In its current type state, createAnimatedComponent cannot create generic components.
     <AnimatedFlatList
       ref={ref}
-      {...restProps}
+      {...copiedProps}
```

Comment:
Instead of copying, we could just also pass it here; 
```suggestion
      {...restProps}
      scrollEventThrottle={restProps.scrollEventThrottle ?? 1}
```
no?